pages/tutorial/simulations.md

```diff
@@ -89,7 +89,25 @@ incarnation: protelis
 
 #### The `seeds` key
 
-> To be written
+The `seed` section may contains two optional values
+
+**Example**
+{% highlight yaml %}
+variables:
```

Comment:
Variables have not yet been introduced at this point. Please use just an integer instead (also, makes the example simpler)